### DISCUSSION TITLE
Arm-based hosted runners are coming to GitHub Actions! 🚢

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#73040] [Actions]
![Screenshot 2023-10-31 at 5 10 17 PM](https://github.com/community/community/assets/95834130/91fcd603-9825-4a55-820f-434de60cfe51)

## Accelerate your CI/CD with Arm-based hosted runners in GitHub Actions

GitHub is introducing new capabilities that use Arm architecture to boost software development speed. These integrated runners, powered by Arm-based Ampere Altra processors, seamlessly work with GitHub Actions. They benefit developers across automotive, IoT, and industrial sectors. These enhancements also promote sustainability and cost efficiency while reducing carbon footprints. **The Arm-based GitHub-hosted runners are set to enter private beta in January 2024.**

For more details, **[see the detailed changelog post](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)**.

#### Interested? **[Click here](https://resources.github.com/devops/accelerate-your-cicd-with-arm-and-gpu-runners-in-github-actions/)** to join the waitlist for the private beta. 🚀

**Comment by mpriscella** [comment]
I tried signing up for the private beta waitlist last night and this morning, but the form is failing validation on submit.

<img width="433" alt="Screenshot 2023-10-31 at 9 32 55 AM" src="https://github.com/community/community/assets/384382/33b66393-7adc-4687-bd69-fd6d86389aad">

Any help would be appreciated, I've been looking forward to Arm-based hosted runners for a while now!

**Comment by Akash1134** [comment]
Hi @mpriscella , Thanks for reporting this and sorry about the hassle. I have raised this to my team internally and will update you with the next steps soon.

**Comment by Akash1134** [comment]
Hi @mpriscella , this was an issue with validation on the backend that we've already fixed. Even when the form errored out - it might've still recorded your submission. But we want to ensure you're on the waitlist. Could you please fill in the form again and let us know if it works now? Thanks!

**Reply by mpriscella** [reply]
Great, thank you! I was able to sign up successfully and am looking forward to hearing more about the beta.

**Comment by ghost** [comment]
From the [waitlist pages](https://resources.github.com/devops/accelerate-your-cicd-with-arm-and-gpu-runners-in-github-actions/)
> **What kind of subscription do I need to access these runners?**
>
> These runners are available to those using the Team or Enterprise Cloud plans.

Is there any plans to extend ARM runners support for free subscriptions public repositories?

**Comment by James-Mart** [comment]
I joined the waitlist a while ago, and haven't heard anything. 
1. Did the private Beta start in January as planned? 
2. How long should I expect to wait before I'm invited to participate in the Beta? Our team will greatly benefit from the arm runners.

**Reply by flammino** [reply]
following

**Comment by terusaku** [comment]
I submitted the waitlist last year on my Organization, but recieved nothing.
I want GitHub to update this status.

Is there anyone use Arm-based GitHub-hosted runners ?

**Reply by cristianrat** [reply]
I also didn't get a confirmation. We'll just have to wait and see.
in the meantime, I'm using dockerx building for arm (WITH caching, otherwise it's soooo slow)

**Reply by terusaku** [reply]
Yes, now I also used [cache](https://github.com/actions/cache), [setup-buildx-action](https://github.com/docker/setup-buildx-action) and [docker/setup-qemu-action](https://github.com/docker/setup-qemu-action).

But if no cached, Nuxt app of production mode consumed 1h20m for build.
I turned linux/amd64 of CPU architecture, then build time shrinked to 3m...

Until the day comes, I'll use linux/amd64 build.

**Reply by ghost** [reply]
Now we are using `runs-on: buildjet-4vcpu-ubuntu-2204-arm` as a straight-forward replacement for ARM, but we are limited to the Linux OS only runners - waiting for a native Linux/Windows ARM ones.

**Comment by ghost** [comment]
[GitHub Actions: Arm based hosted runners for OSS #970](https://github.com/github/roadmap/issues/970)

**Comment by ghost** [comment]
https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta
> Arm runners are available to customers on our Team and Enterprise Cloud plans. **We expect to begin offering Arm runners for open source and personal accounts by the end of the year.**
